$OpenBSD: patch-src_main_c,v 1.1 2015/06/19 12:44:49 ajacoutot Exp $

main.c:317: warning: dereferencing 'void *' pointer
main.c:317: error: request for member '_file' in something not a structure or union

--- src/main.c.orig	Fri Jun 19 14:37:26 2015
+++ src/main.c	Fri Jun 19 14:37:27 2015
@@ -55,7 +55,7 @@ pcre_extra *hints;
 char *boxname, *outboxname, *pipecmd, *tmpfilename;
 int maildir_count = 0;
 int count = 0;
-void *tmpp;
+FILE *tmpp;
 checksum_t *cs;
 
 int
